subgraphs/uniswap-forks/package-lock.json

```diff
@@ -11,7 +11,6 @@
         "npx": "^10.2.2"
       },
       "devDependencies": {
```

Comment:
If you got this by adding mustache to this `package.json`. We recommend to add it globally to prevent this: https://github.com/messari/subgraphs/blob/master/docs/SETUP.md#installation